CHANGELOG.md

```diff
@@ -0,0 +1,45 @@
+Changelog
+====
+
+Below you can changelogs for the trunk version of remoting.
```

Comment:
Please add "view" so this reads, "Below you can view changelogs for the trunk version of remoting."